accounts/hd_test.go

```diff
@@ -30,43 +30,43 @@ func TestHDPathParsing(t *testing.T) {
 		output DerivationPath
 	}{
 		// Plain absolute derivation paths
-		{"m/44'/60'/0'/0", DerivationPath{0x80000000 + 44, 0x80000000 + 60, 0x80000000 + 0, 0}},
-		{"m/44'/60'/0'/128", DerivationPath{0x80000000 + 44, 0x80000000 + 60, 0x80000000 + 0, 128}},
-		{"m/44'/60'/0'/0'", DerivationPath{0x80000000 + 44, 0x80000000 + 60, 0x80000000 + 0, 0x80000000 + 0}},
-		{"m/44'/60'/0'/128'", DerivationPath{0x80000000 + 44, 0x80000000 + 60, 0x80000000 + 0, 0x80000000 + 128}},
-		{"m/2147483692/2147483708/2147483648/0", DerivationPath{0x80000000 + 44, 0x80000000 + 60, 0x80000000 + 0, 0}},
-		{"m/2147483692/2147483708/2147483648/2147483648", DerivationPath{0x80000000 + 44, 0x80000000 + 60, 0x80000000 + 0, 0x80000000 + 0}},
+		{"m/44'/61'/0'/0", DerivationPath{0x80000000 + 44, 0x80000000 + 61, 0x80000000 + 0, 0}},
+		{"m/44'/61'/0'/128", DerivationPath{0x80000000 + 44, 0x80000000 + 61, 0x80000000 + 0, 128}},
+		{"m/44'/61'/0'/0'", DerivationPath{0x80000000 + 44, 0x80000000 + 61, 0x80000000 + 0, 0x80000000 + 0}},
+		{"m/44'/61'/0'/128'", DerivationPath{0x80000000 + 44, 0x80000000 + 61, 0x80000000 + 0, 0x80000000 + 128}},
+		{"m/2147483692/2147483709/2147483648/0", DerivationPath{0x80000000 + 44, 0x80000000 + 61, 0x80000000 + 0, 0}},
```

Comment:
Just a note to self: 
- `fromhex(0x8000003d) => 2147483709`

https://github.com/satoshilabs/slips/blob/master/slip-0044.md